<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGN_PlayerControl</class>
 <widget class="QWidget" name="PGN_PlayerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_title">
     <property name="text">
      <string>Match Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_date">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_result">
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_first">
        <property name="toolTip">
         <string>First Move</string>
        </property>
        <property name="text">
         <string>|&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_prev">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Previous Move</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_moveNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select Move Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Next Move</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_last">
        <property name="toolTip">
         <string>Last Move</string>
        </property>
        <property name="text">
         <string>&gt;|</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="pgn_view"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
